### PR TITLE
Tofe/fixes

### PR DESCRIPTION
**Pull request by Tofee**

### CODE REVIEW

File: framework/build/enyo-build.js [hunk 1]
```diff
@@ -5872,8 +5866,7 @@ return enyo.fetchConfigFile("$enyo/../framework_config.json");
 }, enyo.fetchDeviceInfo = function() {
 return window.PalmSystem ? JSON.parse(PalmSystem.deviceInfo) : undefined;
 }, enyo.requiresWindow(function() {
-if ((window.PalmSystem) && (window.PalmSystem.getResource) && (!window.palmGetResource)) {window.palmGetResource = PalmSystem.getResource;}
-window.addEventListener("load", enyo.ready, !1), window.addEventListener("resize", enyo.sendOrientationChange, !1), Mojo = window.Mojo || {}, Mojo.lowMemoryNotification = function(a) {
+window.addEventListener("load", enyo.ready, !1), window.addEventListener("resize", enyo.sendOrientationChange, !1), Mojo = window.Mojo || {}, window.PalmSystem && window.PalmSystem.getResource && !window.palmGetResource && (window.palmGetResource = PalmSystem.getResource), Mojo.lowMemoryNotification = function(a) {
```
Comment (Herrie82) [inline framework/build/enyo-build.js:5869]: This looks a bit off but might be wrong

File: framework/build/enyo-build.js [hunk 2]
```diff
@@ -7611,7 +7604,7 @@ this.$.view.callBrowserAdapter(a, b);
 webviewClick: function(a, b, c) {
 c && (c.element == "SELECT" ? this._selectRect = c.bounds : this._selectRect = null, this.doClick(b, c));
 }
-}), window.PalmSystem || (enyo.WebView = enyo.Iframe);
+}), !window.PalmSystem && !enyo.args.haveBrowserAdapter && (enyo.WebView = enyo.Iframe);
```
Comment (Herrie82) [inline framework/build/enyo-build.js:7607]: Here is some change too

File: framework/build/enyo-build.js (hunk 1 above)
Comment (Tofee) [reply framework/build/enyo-build.js:5869]: For this one, I think it's just that the lines were not in the same order in the minified version and in the source code. But looking at the code, the behavior should be correct anyway.

File: framework/build/enyo-build.js (hunk 2 above)
Comment (Tofee) [reply framework/build/enyo-build.js:7607]: This corresponds to fix change: https://github.com/webOS-ports/enyo-1.0/commit/563aae1bed99055d988f81380ddf2e7bd25ebb41

**Comment by Herrie82** [comment]
LGTM!

@codepoet80 you have any concerns?

@Tofee got enyo-compress to work, so we can generate the minified version during build, which is a lot nicer vs manually updating both files (and the associated errors with it).

File: framework/build/enyo-build.js [hunk 3]
```diff
@@ -5872,8 +5866,7 @@ return enyo.fetchConfigFile("$enyo/../framework_config.json");
 }, enyo.fetchDeviceInfo = function() {
 return window.PalmSystem ? JSON.parse(PalmSystem.deviceInfo) : undefined;
 }, enyo.requiresWindow(function() {
-if ((window.PalmSystem) && (window.PalmSystem.getResource) && (!window.palmGetResource)) {window.palmGetResource = PalmSystem.getResource;}
```
Comment (codepoet80) [inline framework/build/enyo-build.js:5875]: Why remove the check for palmSystem? Wouldn't this break compat on other devices?

Comment (Herrie82) [reply framework/build/enyo-build.js:5875]: Seems it's not removed, just moved around, see next line. I got confused here too

Comment (codepoet80) [reply framework/build/enyo-build.js:5875]: ah, ok.

**Comment by codepoet80** [comment]
I could find no issues with these changes on modern Android or iOS, Windows, KitKat or iOS 9. Good to go!

**Pull request merged**